motor/motorid/awsec2/metadata_local.go

```diff
@@ -20,13 +23,30 @@ type LocalEc2InstanceMetadata struct {
 
 func (m *LocalEc2InstanceMetadata) Identify() (Identity, error) {
 	metadata := imds.NewFromConfig(m.config)
+	ec2svc := ec2.NewFromConfig(m.config)
 	ctx := context.Background()
 	doc, err := metadata.GetInstanceIdentityDocument(ctx, &imds.GetInstanceIdentityDocumentInput{})
 	if err != nil {
 		return Identity{}, err
 	}
+	filters := []ec2types.Filter{
+		{
+			Name:   aws.String("resource-id"),
+			Values: []string{doc.InstanceID},
+		},
+	}
+	tags, err := ec2svc.DescribeTags(ctx, &ec2.DescribeTagsInput{Filters: filters})
```

Comment:
is this information not available from the metadata service? This looks like it talks to aws?